service-base/src/main/java/org/eclipse/hono/service/registration/RegistrationService.java

```diff
@@ -115,6 +115,10 @@ default void assertRegistration(final String tenantId, final String deviceId, fi
      * Implementing classes should verify, that the gateway is authorized to get an assertion for the device.
      * Such a check might be based on a specific role that the client needs to have or on an
      * explicitly defined relation between the gateway and the device(s).
+     * <br>
+     * In the case of a device configured with multiple <em>via</em> gateways, implementing classes should
+     * update the device's registration information with the given gateway in the form of a <em>last-via</em>
+     * property.
```

Comment:
isn't this relevant to note in the JavaDoc of the other *assertRegistration* variant as well?

Reply:
right, I've added it there as well.